app/javascript/chat/actions/chat_channel_setting_actions.js

```diff
@@ -173,3 +173,24 @@ export async function updateMembershipRole(membershipId, chatChannelId, role) {
 
   return response.json();
 }
+
+/**
+ * Create Chat Channel
+ * @param {string} channelName
+ * @param {string} userNames
+ */
+
+export async function createChannel(channelName, userNames) {
+  const response = await request(`/create_channel`, {
+    method: 'POST',
+    body: {
+      chat_channel: {
+        channel_name: channelName,
+        invitation_usernames: userNames,
+      },
+    },
+    credentials: 'same-origin',
```

Comment:
refactor (non-blocking): This header is not necessary as the `request` utility function handles this by default. See https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js#L47